### ISSUE TITLE
`this.context.freeStyle` is undefined

### ISSUE THREAD

**Issue by namuol** [blakeembrey/react-free-style#6]
(Note: I'm using the `style-element` branch)

When using free-style in my app, I get a fatal exception here:

```js
StyleElement.prototype.componentWillMount = function () {
    if (ExecutionEnvironment.canUseDOM) {
        this.context.freeStyle.addChangeListener(this.onChange); // this.context.freeStyle is undefined
    }
};
```

-----

I'll try to re-produce this in a minimal project and report back.

**Comment by blakeembrey** [comment]
Note: I just updated the title, since I merged `style-element` to master.

Have you tried with the latest React version? Looking forward to seeing a smaller example that can reproduce since the `context` is still pretty new and I'm curious where it's losing the reference. Is there anywhere in the app being rendered outside the React lifecycle?

**Comment by blakeembrey** [comment]
Looks like it's fixed with `0.5.x`. Let me know if you have any more issues :smile:
